tools/util/injectorutil.go

```diff
@@ -728,9 +727,9 @@ func CheckPendingActionList(
 						retErr = err
 						return false
 					}
-					executoraddr, err := address.FromBytes(selp.SrcPubkey().Hash())
-					if err != nil {
-						retErr = err
+					executoraddr := selp.SrcPubkey().Address()
+					if executoraddr != nil {
```

Comment:
== nil
this is why test failed

Reply:
also pls fix @CoderZhi 's comments above

Reply:
ahh.. I missed it. fixed. == nil

Reply:
> also fix @CoderZhi 's comments above

changes done